OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -66,6 +66,15 @@ public class MobileInfo : IMobileInfo, IOccupySpaceInfo, IFacingInfo, UsesInit<F
 		[Desc("Can the actor be ordered to move in to shroud?")]
 		public readonly bool MoveIntoShroud = true;
 
+		[Desc("Randomly select a sound from this list to play when this actor beings moving.")]
+		public readonly string[] Sounds = { };
+
+		[Desc("Should allied players hear movement sounds?")]
+		public readonly bool AlliesHearSounds = true;
+
+		[Desc("Should enemy players hear movement sounds?")]
```

Comment:
Actually it seems like setting this to true will also make the sound hearable for neutrals.